jujugui/static/gui/src/app/components/check-list-item/check-list-item.js

```diff
@@ -116,6 +132,17 @@ YUI.add('check-list-item', function() {
       }
     },
 
+    _generateExtraInfo: function(extraInfo) {
+      if (!extraInfo || extraInfo === '') {
+        return;
```

Comment:
That puts a null into the element so think its best to leave it consistent with other components